platform/windows/export/export.cpp

```diff
@@ -50,21 +50,24 @@ Error EditorExportPlatformWindows::export_project(const Ref<EditorExportPreset>
 
 	String rcedit_path = EditorSettings::get_singleton()->get("export/windows/rcedit");
 
-	if (rcedit_path == String()) {
-		return OK;
-	}
-
-	if (!FileAccess::exists(rcedit_path)) {
+	if (rcedit_path != String() && !FileAccess::exists(rcedit_path)) {
 		ERR_PRINTS("Could not find rcedit executable at " + rcedit_path + ", aborting.");
 		return ERR_FILE_NOT_FOUND;
 	}
 
+	if (rcedit_path == String()) {
+		// Try to run rcedit from PATH
+		// On non-Windows platforms, this will try to call a launcher script
+		// that uses WINE to run the actual rcedit executable
+		rcedit_path = "rcedit";
```

Comment:
This is not really needed here, you can just check `if (rcedit_path == String)` directly before the `execute` call to do the branching.